VikingQuiz.UI/src/Components/QuizItem/QuizItem.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import './QuizItem.css';
+import ManageQuizComponent from '../MyQuizesPage/ManageQuizComponent/ManageQuizComponent';
+
+
+class QuizItem extends React.Component<any, any>{
+    constructor(props: any) {
+        super(props);
+  
+        this.state = {
+            serverMessage: '',
+            redirect: false,
+            isHidden: true,
+            id: this.props.id,
+            handleChildDelete: this.props.handleChildDelete,
```

Comment:
If you're not going to update this state, you can move it further down along with the other functions